spec/support/shared_examples/operation/errors_localizations.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-RSpec.shared_examples 'errors localizations' do
-  it 'errors localizations exist in i18n' do
-    expect(error_localizations).to be_defined_in_i18n
```

Comment:
Do we need to remove `be_defined_in_i18n` matcher?

Reply:
Think we should remove it becouse we use it nowhere